### ISSUE TITLE
Re-trigger doc builds with recent changes

### ISSUE THREAD

**Issue by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#3136]
Follow-up to #3134

Other packages may have had doc building issues since we adopted DocUploader 1.7.1.

It seems like doc generation only started failing (in some cases?) when I updated the builder to use DocUploader for zipping on June 11. I'm pretty sure we'd have see reports if the unzipping caused the issue.

Find and re-run doc builds for all packages with version changes between June 11 and June 14 7:00 CET when the issue was fixed.

**Comment by finestructure** [comment]
105 potentially affected versions, 80 have doc 404s.

```
❯ rester restfiles/doc-issue-test.restfile
🚀  Resting restfiles/doc-issue-test.restfile ...

🎬  /0xWDG/Admob-SwiftUI/main/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("Vary"): Accept-Encoding, AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("Content-Type"): text/html; charset=utf-8, AnyHashable("Age"): 286, AnyHashable("Server"): cloudflare, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=PQt0gTHtWre1GO8oyVqaJKtJAWK5kMLG6UdEMnyZuMnG4R3MBt2aTsm5hHB89mh%2FM%2B%2FcGkqvujHqBMzq%2Fv%2FREwJkVeaqcIApqzyG5uKHBz0zZ6i0MLqzmyIhCMG4lZ%2F%2F6sRsOsfHjg%3D%3D"}],"group":"cf-nel","max_age":604800}, AnyHashable("cf-ray"): 89391e7e6d4f2a16-CDG, AnyHashable("Content-Encoding"): br, AnyHashable("Cache-Control"): max-age=10800, AnyHashable("cf-cache-status"): HIT, AnyHashable("Date"): Fri, 14 Jun 2024 08:58:32 GMT]
Data:     4995 bytes

❌  /0xWDG/Admob-SwiftUI/main/documentation FAILED : status invalid: (404) is not equal to (200)

...

Response:
Status:   404
Headers:  [AnyHashable("Content-Type"): text/html; charset=utf-8, AnyHashable("Date"): Fri, 14 Jun 2024 09:00:12 GMT, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=POOwFXpQacbnU5ikEtmXMQch1%2BYq3dFMYoHW8rHdjNstPbZE53jOgZkW429nEA2s6VDGkB56PnR%2FBOnkZ%2B6NUS4uevAMVnLjlzdf1fLbxrmw74H2dMlfi2x%2BoA%2FFSdxepQMRQp9x2g%3D%3D"}],"group":"cf-nel","max_age":604800}, AnyHashable("cf-ray"): 893920ef799a02c9-CDG, AnyHashable("cf-cache-status"): DYNAMIC, AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("Server"): cloudflare, AnyHashable("Content-Encoding"): br]
Data:     4941 bytes

❌  /xmtp/xmtp-ios/0.11.7/documentation FAILED : status invalid: (404) is not equal to (200)

Executed 105 tests, with 80 failures
```

[doc-issue-test.restfile.zip](https://github.com/user-attachments/files/15835680/doc-issue-test.restfile.zip)

**Comment by finestructure** [comment]
The 80 failures are now down to 13, none of which are associated with the zipping issue. Trimmed and annotated test log:

```
❯ rester restfiles/doc-issue-test.restfile
🎬  /0xWDG/DynamicUI/main/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("Date"): Tue, 25 Jun 2024 09:38:46 GMT, AnyHashable("Content-Type"): text/html; charset=utf-8, AnyHashable("Server"): cloudflare, AnyHashable("Content-Encoding"): br, AnyHashable("cf-cache-status"): DYNAMIC, AnyHashable("cf-ray"): 8993fc929fb56ecc-CDG, AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=LZ7itNfOz7XFl36zHkF9Q7myovdCsMERPmGpErC2gDYuH8qiUFTlbdnnThgn7hYZXDZvmFEi%2FGtbpyLe%2FBUqw7pOq3UIr28P6ZtId6rXIeaJsvRVP8b%2FmqOhwyQ2caXmWAh%2BJZYt2g%3D%3D"}],"group":"cf-nel","max_age":604800}]
Data:     4941 bytes

❌  /0xWDG/DynamicUI/main/documentation FAILED : status invalid: (404) is not equal to (200)
-> misconfigured

🎬  /apple/swift-metrics/main/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("Vary"): Accept-Encoding, AnyHashable("Content-Type"): text/html; charset=utf-8, AnyHashable("Cache-Control"): max-age=10800, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=cgEcGljyXEYaiGgts8prjqbxnQGeBOF15aA9LcfRCc%2B0Dx%2Bk637SHNw85u8Xz96HC5q0kBE%2BC%2BVrCHwMJbq63njmIEEmRB07gjr5O89fdbxOiK5lcDvxyg5%2BQWtFUfGvhceRWsdTyw%3D%3D"}],"group":"cf-nel","max_age":604800}, AnyHashable("Server"): cloudflare, AnyHashable("Date"): Tue, 25 Jun 2024 09:38:52 GMT, AnyHashable("cf-ray"): 8993fcb73b2599c0-CDG, AnyHashable("Content-Encoding"): br, AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("cf-cache-status"): MISS]
Data:     4989 bytes

❌  /apple/swift-metrics/main/documentation FAILED : status invalid: (404) is not equal to (200)
-> known issue

🎬  /apple/swift-syntax/600.0.0-prerelease-2024-06-12/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("Content-Encoding"): br, AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=D14z5XBOUH4GUmxntwGFv7gGZBRGeXh8tD0%2B2mTbed%2Bsebx6Rsizwt7jjNQdr%2Bobe6ZIJ8YUIj7NV6MUOilaV%2FqI4JEIHlAXx2RbhrIKDkmyTu5FlCl%2F5gxMXvNM8SQd1RPVUbpQDA%3D%3D"}],"group":"cf-nel","max_age":604800}, AnyHashable("cf-ray"): 8993fcc6ec326fab-CDG, AnyHashable("cf-cache-status"): DYNAMIC, AnyHashable("Content-Type"): text/html; charset=utf-8, AnyHashable("Server"): cloudflare, AnyHashable("Date"): Tue, 25 Jun 2024 09:38:55 GMT]
Data:     5025 bytes

❌  /apple/swift-syntax/600.0.0-prerelease-2024-06-12/documentation FAILED : status invalid: (404) is not equal to (200)
-> known issue

🎬  /apple/swift-syntax/main/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("cf-cache-status"): DYNAMIC, AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("Content-Encoding"): br, AnyHashable("Server"): cloudflare, AnyHashable("Date"): Tue, 25 Jun 2024 09:38:55 GMT, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=Jrqo7kpjRwHKEXMomq44imSz%2F5Oe7pzieZxAakTRG63TfcSfz3ljTpL8u9uZ1Pa2zYvfOeFe9e2bD0cZO8ktLDI9j4p1mFAWQ%2F5%2BJlGKi8z2N8IMpKBfiNmK3JZfyZooSGIG%2Br%2BZ1A%3D%3D"}],"group":"cf-nel","max_age":604800}, AnyHashable("cf-ray"): 8993fcc8feac701e-CDG, AnyHashable("Content-Type"): text/html; charset=utf-8]
Data:     4950 bytes

❌  /apple/swift-syntax/main/documentation FAILED : status invalid: (404) is not equal to (200)
-> known issue

🎬  /Automattic/Gravatar-SDK-iOS/2.0.1/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("cf-cache-status"): DYNAMIC, AnyHashable("cf-ray"): 8993fcd05f796ef6-CDG, AnyHashable("Content-Encoding"): br, AnyHashable("Date"): Tue, 25 Jun 2024 09:38:56 GMT, AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=qND2g5sTw9QkR2pkN%2FbeubKz8VG3HbjDTVkG%2FXRxD1DVL2iU12zIS0CxeKCaLwq6CJE5i9cyiaH%2BgjWm8hmMTHjp%2B%2FDAQ2BL2gzayUp6l9UCo0PHMGc0oMp86yDs2IzIvkgr74%2BSDA%3D%3D"}],"group":"cf-nel","max_age":604800}, AnyHashable("Server"): cloudflare, AnyHashable("Content-Type"): text/html; charset=utf-8]
Data:     4980 bytes

❌  /Automattic/Gravatar-SDK-iOS/2.0.1/documentation FAILED : status invalid: (404) is not equal to (200)
-> misconfigured
`Error: Unknown option '--include-extended-types'`

🎬  /Automattic/Gravatar-SDK-iOS/trunk/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("Date"): Tue, 25 Jun 2024 09:38:57 GMT, AnyHashable("cf-cache-status"): DYNAMIC, AnyHashable("Content-Encoding"): br, AnyHashable("Content-Type"): text/html; charset=utf-8, AnyHashable("cf-ray"): 8993fcd46e636eed-CDG, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=MdvsXKKTwZqLQOsnmFzRGWK7aNnykA0QmCkZSZSja8Oo3HhTNc38notHplLtiYE5UE5PgV1PPSLS8ZhtTNnuf5VAEooXliqpRkz8N552VM3HrmCuK4Fh0HK5l7BS6fYfKmIpbQBjkQ%3D%3D"}],"group":"cf-nel","max_age":604800}, AnyHashable("Server"): cloudflare, AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}]
Data:     4980 bytes

❌  /Automattic/Gravatar-SDK-iOS/trunk/documentation FAILED : status invalid: (404) is not equal to (200)
-> misconfigured
`Error: Unknown option '--include-extended-types'`

🎬  /daprice/BlurHashViews/v1.0.0/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("Content-Type"): text/html; charset=utf-8, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=8%2FV%2BhJIlnbIBrGsocQvuWzaBdd9rJZ98dvJ0q831KQ8weRCKIkYQz11MnREn2uY%2FSJf0neXymocML7CCiBSZicyXGNctOjPzNKXBIa06gMVJ4g6f5BbrNdVVYXfnrcVMpLdbMhclBw%3D%3D"}],"group":"cf-nel","max_age":604800}, AnyHashable("cf-cache-status"): DYNAMIC, AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("Content-Encoding"): br, AnyHashable("Server"): cloudflare, AnyHashable("cf-ray"): 8993fd0c68796f12-CDG, AnyHashable("Date"): Tue, 25 Jun 2024 09:39:06 GMT]
Data:     4965 bytes

❌  /daprice/BlurHashViews/v1.0.0/documentation FAILED : status invalid: (404) is not equal to (200)
-> available on main, misconfigured for 1.0.0 (package req 6.0, docs built with 5.10)

🎬  /DataDog/dd-sdk-ios/2.13.0/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("cf-cache-status"): DYNAMIC, AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("Content-Encoding"): br, AnyHashable("cf-ray"): 8993fd0ea9f56f18-CDG, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=%2BnivRiM%2BVgH8sHxB1f6R%2Fp9JCDO2ifT8uZMELCezDbGDP7nNCxuL%2FuoZcEJwsVe0JYWlxQFqG5oInhmcYt4h9NnG91ESALY0HYwGeq6d5n5EdjrNJB6kgBqMHPruMjuKsi4vmyvCKw%3D%3D"}],"group":"cf-nel","max_age":604800}, AnyHashable("Server"): cloudflare, AnyHashable("Content-Type"): text/html; charset=utf-8, AnyHashable("Date"): Tue, 25 Jun 2024 09:39:06 GMT]
Data:     4956 bytes

❌  /DataDog/dd-sdk-ios/2.13.0/documentation FAILED : status invalid: (404) is not equal to (200)
-> availabel on develop, misconfigured on 2.13.0
`The workspace named "spi-builder-workspace" does not contain a scheme named "DatadogInternal"`

🎬  /edonv/CollectionView/0.1.4/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("Server"): cloudflare, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=tsrbwoDhAo1AgR7r42MeCsrRmkFP7rzQbJjk6%2FHt9TSFgPVlhcxx%2Fv0ZGoLzFwndFVRKPp8qOLUi1x%2BAKQew%2BAdDLQUFfe5K20gaV0xBAvLM6PC5zw58t9gBth5%2F2dO7EfnMeML8mw%3D%3D"}],"group":"cf-nel","max_age":604800}, AnyHashable("Content-Encoding"): br, AnyHashable("Content-Type"): text/html; charset=utf-8, AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("cf-ray"): 8993fd1f6bacbb6a-CDG, AnyHashable("cf-cache-status"): DYNAMIC, AnyHashable("Date"): Tue, 25 Jun 2024 09:39:09 GMT]
Data:     4959 bytes

❌  /edonv/CollectionView/0.1.4/documentation FAILED : status invalid: (404) is not equal to (200)
-> misconfigured: iOS package building docs on macOS-SwiftPM

🎬  /edonv/CollectionView/main/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("cf-cache-status"): DYNAMIC, AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("cf-ray"): 8993fd21fc74d646-CDG, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=KdPxDPVAHDfsxmigDb75KPAty%2BCt3XNBgXp4XkotEPegzG8Nwi%2FqzYg%2B18yxh8ztUjz1d52Yb1ypA84cCxzqAKNVR1phN%2FCOB1Y2WaHcJHHSZXOdUyKi0nWxPyanL9MMmTBNICk91g%3D%3D"}],"group":"cf-nel","max_age":604800}, AnyHashable("Content-Encoding"): br, AnyHashable("Date"): Tue, 25 Jun 2024 09:39:09 GMT, AnyHashable("Server"): cloudflare, AnyHashable("Content-Type"): text/html; charset=utf-8]
Data:     4956 bytes

❌  /edonv/CollectionView/main/documentation FAILED : status invalid: (404) is not equal to (200)
-> misconfigured: iOS package building docs on macOS-SwiftPM

🎬  /heroesofcode/MagicImages/main/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=7Noebdw45QVyQ7Up5tWNL2L6bh0MbteKUsmuziN7cgdDJWXJNg4L%2F0tToNMSbX%2FZkbmO2NMHH8%2FKH9lKTe8DLHIoUQtya45ZWv3Fcr6R6jMI2QZsqY1D6MsvzuArnhrUAiefmoLuLA%3D%3D"}],"group":"cf-nel","max_age":604800}, AnyHashable("cf-cache-status"): DYNAMIC, AnyHashable("Content-Type"): text/html; charset=utf-8, AnyHashable("Date"): Tue, 25 Jun 2024 09:39:16 GMT, AnyHashable("Server"): cloudflare, AnyHashable("Content-Encoding"): br, AnyHashable("cf-ray"): 8993fd4c1fa62a41-CDG]
Data:     4968 bytes

❌  /heroesofcode/MagicImages/main/documentation FAILED : status invalid: (404) is not equal to (200)
-> misconfigured: iOS package building docs on macOS-SwiftPM

🎬  /pointfreeco/swift-perception/1.2.3/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("Server"): cloudflare, AnyHashable("Date"): Tue, 25 Jun 2024 09:39:46 GMT, AnyHashable("Content-Encoding"): br, AnyHashable("cf-cache-status"): DYNAMIC, AnyHashable("Content-Type"): text/html; charset=utf-8, AnyHashable("cf-ray"): 8993fe059d682155-CDG, AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=0dafAieZA4mIZs%2BWQQ3lLl3N%2F3O6T3N380VADIJC0L68IAFBbC8LJtQm7q7rXBZy7QFU26tDcL5AdIICaZK7qwxvYwxMZTcdDMw2qKbbi3IjuXX9lTUIQi32zw%2BSxqJl6HQefzYJ6A%3D%3D"}],"group":"cf-nel","max_age":604800}]
Data:     4983 bytes

❌  /pointfreeco/swift-perception/1.2.3/documentation FAILED : status invalid: (404) is not equal to (200)
-> was misconfigured, fixed in 1.2.4 and on main

🎬  /xmtp/xmtp-ios/0.11.7/documentation started ...

Response:
Status:   404
Headers:  [AnyHashable("report-to"): {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=xUiY8JGexmtr%2FA7hEVO681jTJC1WayBs0hsC9RrRetTO10yXal%2B5l8nlEBjx4wTO%2FKfetcET9TrAtDpwJdlXI0IK2nTt1YnaDX1XCSkF7PwM79z9d2XfCknmswSwNXCer0r2RNBauQ%3D%3D"}],"group":"cf-nel","max_age":604800}, AnyHashable("Content-Type"): text/html; charset=utf-8, AnyHashable("Content-Encoding"): br, AnyHashable("Server"): cloudflare, AnyHashable("cf-cache-status"): DYNAMIC, AnyHashable("nel"): {"success_fraction":0,"report_to":"cf-nel","max_age":604800}, AnyHashable("Date"): Tue, 25 Jun 2024 09:40:15 GMT, AnyHashable("cf-ray"): 8993feb9da6dbb60-CDG]
Data:     4941 bytes

❌  /xmtp/xmtp-ios/0.11.7/documentation FAILED : status invalid: (404) is not equal to (200)
-> doc gen failed on 0.11.7 (and latest 0.12.3), unclear why, but succeeded on main

Executed 105 tests, with 13 failures
~/P/S/spi-server on main took 1m29s 
```
